File: measurecompute/DAQ/C#/ULAO03/ULAO03.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using System.Data;

//ULAO03================================================================
//
// File:                         ULAO03.CS
//
// Library Call Demonstrated:    MccBoard.AOut()
//                               MccBoard.BoardConfig.DACUpdateMode()
//                               MccBoard.BoardConfig.DACUpdate()
//
// Purpose:                      Demonstrates difference between DACUpdate.Immdediate
//                               and DACUpdate.OnCommand D/A Update modes
//
// Demonstration:                Delays outputs until user issues update command DACUpdate()
//
// Other Library Calls:          MccService.ErrHandling()
//                               MccBoard.FromEngUnits()
//
// Special Requirements:         Board 0 must support BIDACUPDATEMODE settings,
//                               such as the PCI-DAC6700's.
//
//
// (c) Copyright 2003, Measurement Computing Corp.
// All rights reserved.
//==========================================================================
namespace ULAO03
{
	/// <summary>
	/// Summary description for Form1.
	/// </summary>
	public class frmAOut : Form
	{
		private Label label1;
		private Button btnUpdateDACs;
		private Button btnSendData;
		private Label label2;
		private Label label3;
		private Label label4;
		private Label label5;
		private TextBox txtAOVolts0;
		private TextBox txtAOVolts1;
		private TextBox txtAOVolts2;
		private TextBox txtAOVolts3;
		private TextBox[] _txtAOVolts;
		private RadioButton rdioOnCommand;
		private RadioButton rdioImmediate;
		/// <summary>
		/// Required designer variable.
		/// </summary>
		private System.ComponentModel.Container components = null;

		MccDaq.MccBoard _bd;
		MccDaq.Range _range = MccDaq.Range.Bip10Volts;
		const int AllChannels = -1;  //negative values specify all available devices/channels

		public frmAOut()
		{
			//
			// Required for Windows Form Designer support
			//
			InitializeComponent();

			//  Initiate error handling
			//   activating error handling will trap errors like
			//   bad channel numbers and non-configured conditions.
			//   Parameters:
			//     MccDaq.ErrorReporting.PrintAll :all warnings and errors encountered will be printed
			//     MccDaq.ErrorHandling.StopAll   :if an error is encountered, the program will stop
		
			MccDaq.ErrorInfo ULStat = MccDaq.MccService.ErrHandling(MccDaq.ErrorReporting.PrintAll, MccDaq.ErrorHandling.StopAll);
		

			// Create a new MccBoard object for Board 0
			_bd = new MccDaq.MccBoard(0);

			// Initialize outputs to update on command
			// Parameters
			//	 channel	: D/A channel whose update mode is to be configured. Note
			//				  that negative values selects all channels
			//   DACUpdate.OnCommand : delay D/A output updates from AOut or AOutScan until
			//                         DACUpdate command is issued.
			int channel = AllChannels;
			_bd.BoardConfig.SetDACUpdateMode(channel, (int)MccDaq.DACUpdate.OnCommand);

			//  This gives us access to text boxes via an indexed array
			_txtAOVolts = (new TextBox[] {this.txtAOVolts0, this.txtAOVolts1, this.txtAOVolts2, this.txtAOVolts3});  
		}


		/// <summary>
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if (components != null) 
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}




		private void rdioOnCommand_CheckedChanged(object sender, System.EventArgs e)
		{
			if (rdioOnCommand.Checked)
			{
				// Set DAC Update mode to hold off updating D/A's until command is sent
				// Parameters
				//	 channel	: D/A channel whose update mode is to be configured. Note
				//				  that negative values selects all channels
				//   DACUpdate.OnCommand : delay D/A output updates from AOut or AOutScan until
				//                         DACUpdate command is issued.
				int channel = AllChannels;
				_bd.BoardConfig.SetDACUpdateMode(channel, (int)MccDaq.DACUpdate.OnCommand);
			}
		}



		private void rdioImmediate_CheckedChanged(object sender, System.EventArgs e)
		{
			if (rdioImmediate.Checked)
			{
				// Set DAC Update mode to update immediately upon cbAOut or cbAOutScan
				// Parameters
				//	 channel	: D/A channel whose update mode is to be configured. Note
				//				  that negative values selects all channels
				//   DACUpdate.Immediate : update D/A outputs immediately upon AOut or AOutScan
				int channel = AllChannels;
				_bd.BoardConfig.SetDACUpdateMode(channel, (int)MccDaq.DACUpdate.Immediate);
			}
		}


		private void btnSendData_Click(object sender, System.EventArgs e)
		{
			float volts=0.0F;
			ushort daCounts = 0;
			int chan = 0;
			foreach(TextBox box in _txtAOVolts)
			{
				//get voltage to output
				volts = float.Parse(box.Text);	

				// convert from voltage to binary counts
				_bd.FromEngUnits(_range, volts, out daCounts);

				// load D/A
				_bd.AOut(chan, _range, daCounts);
				++chan;
			}
		}



		private void btnUpdateDACs_Click(object sender, System.EventArgs e)
		{
			// Issue D/A Update command
			_bd.BoardConfig.DACUpdate();
		}



		#region Windows Form Designer generated code
		/// <summary>
		/// Required method for Designer support - do not modify
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
			this.label1 = new System.Windows.Forms.Label();
			this.btnUpdateDACs = new System.Windows.Forms.Button();
			this.btnSendData = new System.Windows.Forms.Button();
			this.label2 = new System.Windows.Forms.Label();
			this.label3 = new System.Windows.Forms.Label();
			this.label4 = new System.Windows.Forms.Label();
			this.label5 = new System.Windows.Forms.Label();
			this.txtAOVolts0 = new System.Windows.Forms.TextBox();
			this.txtAOVolts1 = new System.Windows.Forms.TextBox();
			this.txtAOVolts2 = new System.Windows.Forms.TextBox();
			this.txtAOVolts3 = new System.Windows.Forms.TextBox();
			this.rdioOnCommand = new System.Windows.Forms.RadioButton();
			this.rdioImmediate = new System.Windows.Forms.RadioButton();
			this.SuspendLayout();
			// 
			// label1
			// 
			this.label1.Font = new System.Drawing.Font("Microsoft Sans Serif", 12F, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, ((System.Byte)(0)));
			this.label1.Location = new System.Drawing.Point(24, 16);
			this.label1.Name = "label1";
			this.label1.Size = new System.Drawing.Size(296, 48);
			this.label1.TabIndex = 0;
			this.label1.Text = "Demonstrates DAC Update Modes using AOut";
			this.label1.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
			// 
			// btnUpdateDACs
			// 
			this.btnUpdateDACs.Location = new System.Drawing.Point(192, 72);
			this.btnUpdateDACs.Name = "btnUpdateDACs";
			this.btnUpdateDACs.Size = new System.Drawing.Size(136, 32);
			this.btnUpdateDACs.TabIndex = 1;
			this.btnUpdateDACs.Text = "Update Outputs";
			this.btnUpdateDACs.Click += new System.EventHandler(this.btnUpdateDACs_Click);
			// 
			// btnSendData
			// 
			this.btnSendData.Location = new System.Drawing.Point(32, 72);
			this.btnSendData.Name = "btnSendData";
			this.btnSendData.Size = new System.Drawing.Size(136, 32);
			this.btnSendData.TabIndex = 2;
			this.btnSendData.Text = "Send Data";
			this.btnSendData.Click += new System.EventHandler(this.btnSendData_Click);
			// 
			// label2
			// 
			this.label2.Font = new System.Drawing.Font("Microsoft Sans Serif", 9F, (System.Drawing.FontStyle.Bold | System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, ((System.Byte)(0)));
			this.label2.ForeColor = System.Drawing.SystemColors.HotTrack;
			this.label2.Location = new System.Drawing.Point(8, 112);
			this.label2.Name = "label2";
			this.label2.Size = new System.Drawing.Size(80, 16);
			this.label2.TabIndex = 3;
			this.label2.Text = "Channel 0";
			// 
			// label3
			// 
			this.label3.Font = new System.Drawing.Font("Microsoft Sans Serif", 9F, (System.Drawing.FontStyle.Bold | System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, ((System.Byte)(0)));
			this.label3.ForeColor = System.Drawing.SystemColors.HotTrack;
			this.label3.Location = new System.Drawing.Point(96, 112);
			this.label3.Name = "label3";
			this.label3.Size = new System.Drawing.Size(80, 16);
			this.label3.TabIndex = 4;
			this.label3.Text = "Channel 1";
			// 
			// label4
			// 
			this.label4.Font = new System.Drawing.Font("Microsoft Sans Serif", 9F, (System.Drawing.FontStyle.Bold | System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, ((System.Byte)(0)));
			this.label4.ForeColor = System.Drawing.SystemColors.HotTrack;
			this.label4.Location = new System.Drawing.Point(184, 112);
			this.label4.Name = "label4";
			this.label4.Size = new System.Drawing.Size(80, 16);
			this.label4.TabIndex = 5;
			this.label4.Text = "Channel 2";
			// 
			// label5
			// 
			this.label5.Font = new System.Drawing.Font("Microsoft Sans Serif", 9F, (System.Drawing.FontStyle.Bold | System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, ((System.Byte)(0)));
			this.label5.ForeColor = System.Drawing.SystemColors.HotTrack;
			this.label5.Location = new System.Drawing.Point(272, 112);
			this.label5.Name = "label5";
			this.label5.Size = new System.Drawing.Size(80, 16);
			this.label5.TabIndex = 6;
			this.label5.Text = "Channel 3";
			// 
			// txtAOVolts0
			// 
			this.txtAOVolts0.Location = new System.Drawing.Point(8, 136);
			this.txtAOVolts0.Name = "txtAOVolts0";
			this.txtAOVolts0.Size = new System.Drawing.Size(72, 20);
			this.txtAOVolts0.TabIndex = 7;
			this.txtAOVolts0.Text = "0.00";
			// 
			// txtAOVolts1
			// 
			this.txtAOVolts1.Location = new System.Drawing.Point(96, 136);
			this.txtAOVolts1.Name = "txtAOVolts1";
			this.txtAOVolts1.Size = new System.Drawing.Size(72, 20);
			this.txtAOVolts1.TabIndex = 8;
			this.txtAOVolts1.Text = "0.00";
			// 
			// txtAOVolts2
			// 
			this.txtAOVolts2.Location = new System.Drawing.Point(184, 136);
			this.txtAOVolts2.Name = "txtAOVolts2";
			this.txtAOVolts2.Size = new System.Drawing.Size(72, 20);
			this.txtAOVolts2.TabIndex = 9;
			this.txtAOVolts2.Text = "0.00";
			// 
			// txtAOVolts3
			// 
			this.txtAOVolts3.Location = new System.Drawing.Point(272, 136);
			this.txtAOVolts3.Name = "txtAOVolts3";
			this.txtAOVolts3.Size = new System.Drawing.Size(72, 20);
			this.txtAOVolts3.TabIndex = 10;
			this.txtAOVolts3.Text = "0.00";
			// 
			// rdioOnCommand
			// 
			this.rdioOnCommand.Checked = true;
			this.rdioOnCommand.Location = new System.Drawing.Point(96, 176);
			this.rdioOnCommand.Name = "rdioOnCommand";
			this.rdioOnCommand.Size = new System.Drawing.Size(168, 24);
			this.rdioOnCommand.TabIndex = 11;
			this.rdioOnCommand.TabStop = true;
			this.rdioOnCommand.Text = "Update On Command";
			this.rdioOnCommand.CheckedChanged += new System.EventHandler(this.rdioOnCommand_CheckedChanged);
			// 
			// rdioImmediate
			// 
			this.rdioImmediate.Location = new System.Drawing.Point(96, 208);
			this.rdioImmediate.Name = "rdioImmediate";
			this.rdioImmediate.Size = new System.Drawing.Size(168, 24);
			this.rdioImmediate.TabIndex = 12;
			this.rdioImmediate.Text = "Update Immediately";
			this.rdioImmediate.CheckedChanged += new System.EventHandler(this.rdioImmediate_CheckedChanged);
			// 
			// frmAOut
			// 
			this.AutoScaleBaseSize = new System.Drawing.Size(5, 13);
			this.ClientSize = new System.Drawing.Size(360, 245);
			this.Controls.AddRange(new System.Windows.Forms.Control[] {
																		  this.rdioImmediate,
																		  this.rdioOnCommand,
																		  this.txtAOVolts3,
																		  this.txtAOVolts2,
																		  this.txtAOVolts1,
																		  this.txtAOVolts0,
																		  this.label5,
																		  this.label4,
																		  this.label3,
																		  this.label2,
																		  this.btnSendData,
																		  this.btnUpdateDACs,
																		  this.label1});
			this.Name = "frmAOut";
			this.Text = "Universal Library Analog Output";
			this.ResumeLayout(false);

		}
		#endregion

		/// <summary>
		/// The main entry point for the application.
		/// </summary>
		[STAThread]
		static void Main() 
		{
			Application.Run(new frmAOut());
		}

		
	}
}

File: measurecompute/DAQ/VB.NET/ULAO03.NET/ULAO03.vb
'ULAO03================================================================
'
' File:                         ULAO03.VB
'
' Library Call Demonstrated:    MccBoard.AOut()
'                               MccBoard.BoardConfig.SetDACUpdateMode()
'                               MccBoard.BoardConfig.DACUpdate()
'
' Purpose:                      Demonstrates difference between DACUpdate.Immediate
'                               and DACUpdate.OnCommand D/A Update modes
'
' Demonstration:                Delays outputs until user issues update command 
'                               DACUpdate().
'
' Other Library Calls:          MccService.ErrHandling()
'                               MccBoard.FromEngUnits()
'
' Special Requirements:         Board 0 must support BIDACUPDATEMODE settings,
'                               such as the PCI-DAC6700's.
'
'
' (c) Copyright 2003, Measurement Computing Corp.
' All rights reserved.
'==========================================================================
Option Strict Off
Option Explicit On 
Public Class frmAOut
    Inherits System.Windows.Forms.Form

    'Create a new MccBoard object for Board 0
    Private _bd As MccDaq.MccBoard = New MccDaq.MccBoard(0)
    Private _range As MccDaq.Range = MccDaq.Range.Bip10Volts
    Public txtAOVolts As System.Windows.Forms.TextBox()

    Const AllChannels As Integer = -1


    Private Sub rdioOnCommand_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles rdioOnCommand.CheckedChanged
        Dim channel As Integer

        If (rdioOnCommand.Checked) Then
            ' Set DAC Update mode to hold off updating D/A's until command is sent
            ' Parameters
            '	 channel	: D/A channel whose update mode is to be configured. Note
            '				  that negative values selects all channels
            '   DACUpdate.OnCommand : delay D/A output updates from AOut or AOutScan until
            '                         DACUpdate command is issued.
            channel = AllChannels
            _bd.BoardConfig.SetDACUpdateMode(channel, MccDaq.DACUpdate.OnCommand)
        End If

    End Sub



    Private Sub rdioImmediate_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles rdioImmediate.CheckedChanged
        Dim channel As Integer

        If (rdioImmediate.Checked) Then
            ' Set DAC Update mode to update immediately upon cbAOut or cbAOutScan
            ' Parameters
            '	 channel	: D/A channel whose update mode is to be configured. Note
            '				  that negative values selects all channels
            '   DACUpdate.Immediate : update D/A outputs immediately upon AOut or AOutScan
            channel = AllChannels
            _bd.BoardConfig.SetDACUpdateMode(channel, MccDaq.DACUpdate.Immediate)
        End If

    End Sub



    Private Sub btnSendData_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnSendData.Click
        Dim volts As Single = 0.0F
        Dim daCounts As Integer = 0
        Dim chan As Integer

        For chan = 0 To 3
            'get voltage to output
            volts = Val(txtAOVolts(chan).Text)

            ' convert from voltage to binary counts
            _bd.FromEngUnits(_range, volts, daCounts)

            ' load D/A
            _bd.AOut(chan, _range, daCounts)
        Next chan



    End Sub



    Private Sub btnUpdateDACs_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnUpdateDACs.Click

        ' Issue D/A update command
        _bd.BoardConfig.DACUpdate()

    End Sub



    Private Sub frmAOut_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Dim ULStat As MccDaq.ErrorInfo
        Dim channel As Integer

        ' Initiate error handling
        '  activating error handling will trap errors like
        '  bad channel numbers and non-configured conditions.
        '  Parameters:
        '    MccDaq.ErrorReporting.PrintAll :all warnings and errors encountered will be printed
        '    MccDaq.ErrorHandling.StopAll   :if any error is encountered, the program will stop
        ULStat = MccDaq.MccService.ErrHandling(MccDaq.ErrorReporting.PrintAll, MccDaq.ErrorHandling.StopAll)


        ' Set DAC Update mode to hold off updating D/A's until command is sent
        ' Parameters
        '	 channel	: D/A channel whose update mode is to be configured. Note
        '				  that negative values selects all channels
        '   DACUpdate.OnCommand : delay D/A output updates from AOut or AOutScan until
        '                         DACUpdate command is issued.
        channel = AllChannels
        _bd.BoardConfig.SetDACUpdateMode(channel, MccDaq.DACUpdate.OnCommand)


        ' Attach references of voltage textboxes to array for easier access
        txtAOVolts = New System.Windows.Forms.TextBox(4) {}
        
        txtAOVolts.SetValue(txtAOVolts3, 3)
        txtAOVolts.SetValue(txtAOVolts2, 2)
        txtAOVolts.SetValue(txtAOVolts1, 1)
        txtAOVolts.SetValue(txtAOVolts0, 0)
        
    End Sub


#Region " Windows Form Designer generated code "

    Public Sub New()
        MyBase.New()

        'This call is required by the Windows Form Designer.
        InitializeComponent()

        'Add any initialization after the InitializeComponent() call

    End Sub

    'Form overrides dispose to clean up the component list.
    Protected Overloads Overrides Sub Dispose(ByVal disposing As Boolean)
        If disposing Then
            If Not (components Is Nothing) Then
                components.Dispose()
            End If
        End If
        MyBase.Dispose(disposing)
    End Sub

    'Required by the Windows Form Designer
    Private components As System.ComponentModel.IContainer

    'NOTE: The following procedure is required by the Windows Form Designer
    'It can be modified using the Windows Form Designer.  
    'Do not modify it using the code editor.
    Friend WithEvents label5 As System.Windows.Forms.Label
    Friend WithEvents label4 As System.Windows.Forms.Label
    Friend WithEvents label3 As System.Windows.Forms.Label
    Friend WithEvents label2 As System.Windows.Forms.Label
    Friend WithEvents btnSendData As System.Windows.Forms.Button
    Friend WithEvents btnUpdateDACs As System.Windows.Forms.Button
    Friend WithEvents label1 As System.Windows.Forms.Label
    Friend WithEvents rdioImmediate As System.Windows.Forms.RadioButton
    Friend WithEvents rdioOnCommand As System.Windows.Forms.RadioButton
    Friend WithEvents txtAOVolts3 As System.Windows.Forms.TextBox
    Friend WithEvents txtAOVolts2 As System.Windows.Forms.TextBox
    Friend WithEvents txtAOVolts1 As System.Windows.Forms.TextBox
    Friend WithEvents txtAOVolts0 As System.Windows.Forms.TextBox
    <System.Diagnostics.DebuggerStepThrough()> Private Sub InitializeComponent()
        Me.label5 = New System.Windows.Forms.Label()
        Me.label4 = New System.Windows.Forms.Label()
        Me.label3 = New System.Windows.Forms.Label()
        Me.label2 = New System.Windows.Forms.Label()
        Me.btnSendData = New System.Windows.Forms.Button()
        Me.btnUpdateDACs = New System.Windows.Forms.Button()
        Me.label1 = New System.Windows.Forms.Label()
        Me.rdioImmediate = New System.Windows.Forms.RadioButton()
        Me.rdioOnCommand = New System.Windows.Forms.RadioButton()
        Me.txtAOVolts3 = New System.Windows.Forms.TextBox()
        Me.txtAOVolts2 = New System.Windows.Forms.TextBox()
        Me.txtAOVolts1 = New System.Windows.Forms.TextBox()
        Me.txtAOVolts0 = New System.Windows.Forms.TextBox()
        Me.SuspendLayout()
        '
        'label5
        '
        Me.label5.Font = New System.Drawing.Font("Microsoft Sans Serif", 9.0!, (System.Drawing.FontStyle.Bold Or System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, CType(0, Byte))
        Me.label5.ForeColor = System.Drawing.SystemColors.HotTrack
        Me.label5.Location = New System.Drawing.Point(288, 104)
        Me.label5.Name = "label5"
        Me.label5.Size = New System.Drawing.Size(80, 16)
        Me.label5.TabIndex = 13
        Me.label5.Text = "Channel 3"
        '
        'label4
        '
        Me.label4.Font = New System.Drawing.Font("Microsoft Sans Serif", 9.0!, (System.Drawing.FontStyle.Bold Or System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, CType(0, Byte))
        Me.label4.ForeColor = System.Drawing.SystemColors.HotTrack
        Me.label4.Location = New System.Drawing.Point(200, 104)
        Me.label4.Name = "label4"
        Me.label4.Size = New System.Drawing.Size(80, 16)
        Me.label4.TabIndex = 12
        Me.label4.Text = "Channel 2"
        '
        'label3
        '
        Me.label3.Font = New System.Drawing.Font("Microsoft Sans Serif", 9.0!, (System.Drawing.FontStyle.Bold Or System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, CType(0, Byte))
        Me.label3.ForeColor = System.Drawing.SystemColors.HotTrack
        Me.label3.Location = New System.Drawing.Point(112, 104)
        Me.label3.Name = "label3"
        Me.label3.Size = New System.Drawing.Size(80, 16)
        Me.label3.TabIndex = 11
        Me.label3.Text = "Channel 1"
        '
        'label2
        '
        Me.label2.Font = New System.Drawing.Font("Microsoft Sans Serif", 9.0!, (System.Drawing.FontStyle.Bold Or System.Drawing.FontStyle.Underline), System.Drawing.GraphicsUnit.Point, CType(0, Byte))
        Me.label2.ForeColor = System.Drawing.SystemColors.HotTrack
        Me.label2.Location = New System.Drawing.Point(24, 104)
        Me.label2.Name = "label2"
        Me.label2.Size = New System.Drawing.Size(80, 16)
        Me.label2.TabIndex = 10
        Me.label2.Text = "Channel 0"
        '
        'btnSendData
        '
        Me.btnSendData.Location = New System.Drawing.Point(48, 64)
        Me.btnSendData.Name = "btnSendData"
        Me.btnSendData.Size = New System.Drawing.Size(136, 32)
        Me.btnSendData.TabIndex = 9
        Me.btnSendData.Text = "Send Data"
        '
        'btnUpdateDACs
        '
        Me.btnUpdateDACs.Location = New System.Drawing.Point(208, 64)
        Me.btnUpdateDACs.Name = "btnUpdateDACs"
        Me.btnUpdateDACs.Size = New System.Drawing.Size(136, 32)
        Me.btnUpdateDACs.TabIndex = 8
        Me.btnUpdateDACs.Text = "Update Outputs"
        '
        'label1
        '
        Me.label1.Font = New System.Drawing.Font("Microsoft Sans Serif", 12.0!, System.Drawing.FontStyle.Bold, System.Drawing.GraphicsUnit.Point, CType(0, Byte))
        Me.label1.Location = New System.Drawing.Point(40, 8)
        Me.label1.Name = "label1"
        Me.label1.Size = New System.Drawing.Size(296, 48)
        Me.label1.TabIndex = 7
        Me.label1.Text = "Demonstrates DAC Update Modes using AOut"
        Me.label1.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
        '
        'rdioImmediate
        '
        Me.rdioImmediate.Location = New System.Drawing.Point(104, 200)
        Me.rdioImmediate.Name = "rdioImmediate"
        Me.rdioImmediate.Size = New System.Drawing.Size(168, 24)
        Me.rdioImmediate.TabIndex = 15
        Me.rdioImmediate.Text = "Update Immediately"
        '
        'rdioOnCommand
        '
        Me.rdioOnCommand.Checked = True
        Me.rdioOnCommand.Location = New System.Drawing.Point(104, 168)
        Me.rdioOnCommand.Name = "rdioOnCommand"
        Me.rdioOnCommand.Size = New System.Drawing.Size(168, 24)
        Me.rdioOnCommand.TabIndex = 14
        Me.rdioOnCommand.TabStop = True
        Me.rdioOnCommand.Text = "Update On Command"
        '
        'txtAOVolts3
        '
        Me.txtAOVolts3.Location = New System.Drawing.Point(292, 128)
        Me.txtAOVolts3.Name = "txtAOVolts3"
        Me.txtAOVolts3.Size = New System.Drawing.Size(72, 20)
        Me.txtAOVolts3.TabIndex = 19
        Me.txtAOVolts3.Text = "0.00"
        '
        'txtAOVolts2
        '
        Me.txtAOVolts2.Location = New System.Drawing.Point(204, 128)
        Me.txtAOVolts2.Name = "txtAOVolts2"
        Me.txtAOVolts2.Size = New System.Drawing.Size(72, 20)
        Me.txtAOVolts2.TabIndex = 18
        Me.txtAOVolts2.Text = "0.00"
        '
        'txtAOVolts1
        '
        Me.txtAOVolts1.Location = New System.Drawing.Point(116, 128)
        Me.txtAOVolts1.Name = "txtAOVolts1"
        Me.txtAOVolts1.Size = New System.Drawing.Size(72, 20)
        Me.txtAOVolts1.TabIndex = 17
        Me.txtAOVolts1.Text = "0.00"
        '
        'txtAOVolts0
        '
        Me.txtAOVolts0.Location = New System.Drawing.Point(28, 128)
        Me.txtAOVolts0.Name = "txtAOVolts0"
        Me.txtAOVolts0.Size = New System.Drawing.Size(72, 20)
        Me.txtAOVolts0.TabIndex = 16
        Me.txtAOVolts0.Text = "0.00"
        '
        'frmAOut
        '
        Me.AutoScaleBaseSize = New System.Drawing.Size(5, 13)
        Me.ClientSize = New System.Drawing.Size(392, 237)
        Me.Controls.AddRange(New System.Windows.Forms.Control() {Me.txtAOVolts3, Me.txtAOVolts2, Me.txtAOVolts1, Me.txtAOVolts0, Me.rdioImmediate, Me.rdioOnCommand, Me.label5, Me.label4, Me.label3, Me.label2, Me.btnSendData, Me.btnUpdateDACs, Me.label1})
        Me.Name = "frmAOut"
        Me.Text = "Universal Library Analog Output"
        Me.ResumeLayout(False)

    End Sub

#End Region


End Class
